### PR TITLE
Added Articulating Raspberry Pi 7" Display Mount

### PR DESCRIPTION
**Pull request by revnull-src**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Articulating Raspberry Pi 7" Display Mount

#### How was it tested?
Tested by mounting display to frame and testing full range of articulation.

#### Any background context you want to provide?
A giant printer (2.4 350) should have a giant screen. :)

#### Screenshots (if appropriate)
Pictures included in the "Images" subdir.

#### Further notes
None at this time.

### CODE REVIEW

File: printer_mods/revnull/rpi_7in_display_mount/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* 8 x M3x8 (SHCS)
+* 2 x M4x8 (SHCS)
+* 2 x M4 T/Hammer Nuts
+* Zip ties
+
+**Optional Display Case Back:**
+* [luc_e](https://www.thingiverse.com/thing:1585924)
+* [mjcopper](https://www.thingiverse.com/thing:4574770)
+
+Assemble the parts in the following orientation making sure not to overtighten/scrip the screws. The movement should be smooth, but tight enough to hold position.
```
Comment (KiloQubit) [inline printer_mods/revnull/rpi_7in_display_mount/README.md:24]: Replace `scrip` with `strip`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Just a minor correction in the README below.

File: printer_mods/revnull/rpi_7in_display_mount/README.md (hunk 1 above)
Comment (revnull-src) [reply printer_mods/revnull/rpi_7in_display_mount/README.md:24]: Thanks for the typo catch.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Comment by Fr33man85** [comment]
Great Display mount, but with one major issue: not usable, if you using the front doors with upper and lower handles :(

is it possible to get the CAD-files?


20210908_195603_1024x1024_500KB
20210908_195610_1024x1024_500KB
20210908_195614_1024x1024_500KB
20210908_195625_1024x1024_500KB

**Pull request merged**